doc/ug/dv_methodology/index.md

```diff
@@ -262,6 +263,10 @@ In DV, we follow the same assertion methodology as indicated in the [design meth
 Wherever possible, the assertion monitors developed for FPV are reused in UVM testbenches when running dynamic simulations.
 An example of this is the [TLUL Protocol Checker]({{< relref "hw/ip/tlul/doc/TlulProtocolChecker.md" >}}).
 
+Unlike design assertions, in DV assertions are typically created within SV interfaces bound to the DUT.
+This way assertions and any collateral code don't affect the design, and can reach any internal design signal if needed.
+For an example of this see the [clkmgr assertions]({{< relref "hw/ip/clkmgr/dv/sva" >}}.
```

Comment:
relref doesn't work for this, so the CI fails due to document render errors.

You may need to use the full path like this one.
`['common_cov_excl.cfg'](https://github.com/lowRISC/opentitan/blob/master/hw/dv/tools/vcs/common_cov_excl.cfg)`